imgui_demo.cpp

```diff
@@ -537,14 +537,14 @@ void ImGui::ShowDemoWindow(bool* p_open)
 
             if (ImGui::TreeNode("Word Wrapping"))
             {
-				ImGui::SetTextAlignment(ImGuiTextAlignment_Right);
+                ImGui::PushTextAlignment(1.0);
                 // Using shortcut. You can use PushTextWrapPos()/PopTextWrapPos() for more flexibility.
                 ImGui::TextWrapped("[Right Align] This text should automatically wrap on the edge of the window. The current implementation for text wrapping follows simple rules suitable for English and possibly other languages.");
-				ImGui::Spacing();
-				ImGui::SetTextAlignment(ImGuiTextAlignment_Center);
-				ImGui::TextWrapped("[Center Align] This text should automatically wrap on the edge of the window. The current implementation for text wrapping follows simple rules suitable for English and possibly other languages.");
-				ImGui::SetTextAlignment();
-				ImGui::Spacing();
+                ImGui::Spacing();
+                ImGui::PushTextAlignment(0.5);
+                ImGui::TextWrapped("[Center Align] This text should automatically wrap on the edge of the window. The current implementation for text wrapping follows simple rules suitable for English and possibly other languages.");
```

Comment:
Use 0.5f for floating point value, 0.5 is a double which will warn with some compiler settings.

Reply:
noted, I will update in the next commit.